mathext/ell_carlson.go

```diff
@@ -0,0 +1,145 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mathext
+
+import (
+	"math"
+)
+
+// EllipticRF computes the symmetric elliptic integral R_F(x,y,z):
+//
+//	R_F(x,y,z) = (1/2)\int_{0}^{\infty}{1/s(t)} dt,
+//	s(t) = \sqrt{(t+x)(t+y)(t+z)}.
+//
```

Comment:
The rules when this returns NaN are slightly complicated but still they should be mentioned here somehow.